src/components/map/Map.tsx

```diff
@@ -78,6 +103,66 @@ export default function Map() {
         zoomOffset={-1}
         zIndex={mapZIndex.Labels}
       />
+      {state.showEstimatePins && state.explore.records.map((record, idx) => 
+        {
+          if(record.pin_latitude && record.pin_longitude){
+            return (
+              <Marker 
+                eventHandlers={{
+                  click: (e) => {
+                    (e.target as any).openPopup();
+                  },
+                }}
+                icon={record.estimate_grade === "National" ? nationalMarker : (record.estimate_grade === "Regional" ? regionalMarker : localMarker)}
+                key={`marker-${idx}`} 
+                position={[record.pin_latitude, record.pin_longitude]}
+              >
+                <Popup autoClose={false} className="pin-popup">
```

Comment:
In general let's use our translation system here instead of hardcoded strings

Reply:
Actually, would it be easier for you to deal with incorporating translations here @ewanmay given that you're currently working on translations as a whole?

Reply:
Yeah I can do that